quiche/src/lib.rs

```diff
@@ -1878,6 +1895,15 @@ impl Connection {
             stopped_stream_remote_count: 0,
         };
 
+        // Enable path MTU Discovery and start probing with the largest datagram
+        // size
+        if config.pmtud {
+            let path_pmtud = &mut conn.paths.get_mut(active_path_id)?.pmtud;
+            path_pmtud.should_probe(config.pmtud);
+            path_pmtud.set_probe_size(config.max_send_udp_payload_size);
```

Comment:
This made me wonder, in this PR does the maximum probe size get updated when we know the`max_udp_payload_size` TP from the peer (either the default value, or one that they set specifically)?